### ISSUE TITLE
Error Installing Requirements on Kaggle and Colab

### ISSUE THREAD

**Issue by realonbebeto** [AarohiSingla/Mask-R-CNN-using-Tensorflow2#1]
I have challenges installing tensorflow==2.4.0, scikit-image==0.16.2 and keras==2.4.0 even locally on a virtualenv

**Comment by DavidFlowbender** [comment]
I'm stucked too at importing the dependencies
